src/shared/common/forms/composite/bulk-document-add.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
Try reusing `document-add.jsx` and simply changing the properties in the `.json` file to overwrite the bits you want to change.